packages/plugin-ext/src/hosted/node/scanners/scanner-theia.ts

```diff
@@ -94,17 +98,70 @@ export class TheiaPluginScanner implements PluginScanner {
             contributions.grammars = grammars;
         }
 
+        if (rawPlugin.contributes!.viewsContainers) {
+            contributions.viewsContainers = {};
+
+            for (let location in rawPlugin.contributes.viewsContainers!) {
```

Comment:
is it possible to use arrays.map or array.forEach ?

Reply:
Yes, but using Object.keys(..).forEach(..)
I will rework it